conf/wdio-browserstack-local.conf.js

```diff
@@ -0,0 +1,55 @@
+var defaults = require("./wdio.conf.js");
+var browserstack = require('browserstack-local');
+var _ = require("lodash");
+
+var overrides = {
+  user: process.env.BROWSERSTACK_USERNAME || 'BROWSERSTACK_USERNAME',
+  key: process.env.BROWSERSTACK_ACCESS_KEY || 'BROWSERSTACK_ACCESS_KEY',
+  specs: [
+    './test/specs/local/orders-local.spec.js'
```

Comment:
use the "e2e.spec.js" file for local as well rather than creating a new spec file